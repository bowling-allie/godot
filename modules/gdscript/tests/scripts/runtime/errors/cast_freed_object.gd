#debug-only: Run-time freed instance checks are disabled in release

func test():
	var node := Node.new()
	node.free()
	print(node as Node2D)
