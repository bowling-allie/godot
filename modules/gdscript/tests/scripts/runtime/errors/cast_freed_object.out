GDTEST_RUNTIME_ERROR
>> SCRIPT ERROR at runtime/errors/cast_freed_object.gd:6 on test(): Trying to cast a freed object.
